source/common/network/utility.cc

```diff
@@ -132,14 +133,20 @@ Address::InstanceConstSharedPtr Utility::parseInternetAddressNoThrow(const std::
   if (inet_pton(AF_INET, ip_address.c_str(), &sa4.sin_addr) == 1) {
     sa4.sin_family = AF_INET;
     sa4.sin_port = htons(port);
-    return std::make_shared<Address::Ipv4Instance>(&sa4);
+    StatusOr<Address::InstanceConstSharedPtr> error_or_address =
+        Address::InstanceFactory::createInstancePtr<Address::Ipv4Instance>(&sa4);
+    RELEASE_ASSERT(error_or_address.ok(), "");
```

Comment:
Use error_or_address.message() here as the second arg to RELEASE_ASSERT, here and elsewhere.